serverless/createsu.py

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python
+import os
+from django.contrib.auth.models import User
+
+
+def handler(self, *args, **options):
+    if not User.objects.filter(username='root').exists():
```

Comment:
Just for extra paranoia would you make both username and password configurable by env var and remove the default for the password.